rewrite-maven/src/test/java/org/openrewrite/maven/AddRuntimeConfigTest.java

```diff
@@ -24,7 +24,11 @@
 import org.openrewrite.test.RewriteTest;
 import org.openrewrite.test.SourceSpecs;
 
-import static org.openrewrite.maven.AddRuntimeConfig.*;
+import static org.openrewrite.maven.AddRuntimeConfig.JVM_CONFIG_FILENAME;
+import static org.openrewrite.maven.AddRuntimeConfig.JVM_CONFIG_PATH;
+import static org.openrewrite.maven.AddRuntimeConfig.MAVEN_CONFIG_FILENAME;
+import static org.openrewrite.maven.AddRuntimeConfig.MAVEN_CONFIG_PATH;
+import static org.openrewrite.maven.AddRuntimeConfig.Separator;
```

Comment:
Similarly, here we have a mix of 4 static fields and an enum, that together add up to 5+ elements for a wildcard import.
```suggestion
import static org.openrewrite.maven.AddRuntimeConfig.*;
```